src/pages/Login/Login.layout.vue

```diff
@@ -55,21 +55,51 @@
         </div>
       </q-form>
       <div class="login__other_links">
-        <a class="login__forgotten_link" href="mailto:adidperu@gmail.com"
-          >¿Olvidaste tu contraseña?</a
-        >
-        <a class="login__get_account" href="mailto:adidperu@gmail.com">
+        <button @click="openRecoverPasswordModal" class="login__forgotten_link">
+          ¿Olvidaste tu contraseña?
+        </button>
+        <button @click="openRecoverPasswordModal" class="login__get_account">
           Crear cuenta
-        </a>
+        </button>
       </div>
     </main>
   </div>
 </template>
 
 <script setup lang="ts">
 import didLogo from 'src/assets/logos/didperu.svg';
-import { QInputProps } from 'quasar';
+import infoIcon from 'src/assets/icons/info-svgrepo-com.svg';
+import { QInputProps, copyToClipboard, Dialog, Notify } from 'quasar';
 
+const openRecoverPasswordModal = () => {
+  const email = 'adidperu@gmail.com';
+  Dialog.create({
+    title: `<img class="infoIcon" src="${infoIcon}" alt="DID Logo" />`,
+    message: `
+    <div class="text-center">
+      Si deseas recuperar tu cuenta o crear una puedes ponerte en contacto con nosotros y te ayudaremos haz saber tu peticion a este email:
+      <div class="email_link" color="brand">${email}<div>
+    </div>`,
+    html: true,
+    cancel: {
+      textColor: 'primary',
+      color: 'secondary',
+      label: 'Entendido',
+    },
+    ok: {
+      textColor: 'secondary',
+      color: 'primary',
+      label: 'Copiar',
```

Comment:
nit: I think a plus here could be clarified to the user what will copy when clicks in Copiar, I propose adding a copy icon about the email or maybe another indicator

Reply:
what about now

Reply:
Looks good, good work!